nextcord/state.py

```diff
@@ -1798,6 +1802,19 @@ def parse_guild_stickers_update(self, data) -> None:
         guild.stickers = tuple([self.store_sticker(guild, d) for d in data["stickers"]])
         self.dispatch("guild_stickers_update", guild, before_stickers, guild.stickers)
 
+    def parse_application_command_permissions_update(
+        self, data: GuildApplicationCommandPermissionsPayload
+    ) -> None:
+        guild = self._get_guild(int(data["guild_id"]))
```

Comment:
I understand this logic is used in almost every event, but the resulting `guild` object seems to not be used, you should pass the state (and get rid of this) or the guild object to make this useful.